images/win/Windows2016-Azure.json

```diff
@@ -532,7 +532,7 @@
         {
             "type": "powershell",
             "scripts":[
-                "{{ template_dir }}/scripts/Installers/Install-MinGW.ps1"
+                "{{ template_dir }}/scripts/Installers/Install-Mingw64.ps1"
```

Comment:
The naming is a little confusing I agree - I think the name is *technically* "MinGW-w64" {though I confess I tend to use "Mingw-w64" - and so, sometimes, does the project itself} see, e.g.: http://mingw-w64.org/doku.php/start . Confusingly the project does use `mingw64` and `mingw32` as sub-directories/system roots in the file-system into which MSYS2 is installed - but those are used to segregate the two sets of objects from the `mingw-w64-i686-xxx`  and `mingw-w64-x86_64-xxx` package-sets {and also the packages from the third repository, the less-Windows/more P.O.S.I.X. MSYS2 core itself}.

Unless there is a compelling reason not to include another hyphen I'd gently suggest the PowerShell script to be `Install-Mingw-w64.ps1`

Reply:
@SlySven, Thank you for description .PowerShell uses a "verb-noun" naming system and we are trying to follow this naming

Reply:
Ah, I get you.